libutil/dr_config.c

```diff
@@ -271,7 +271,7 @@ add_extra_option(opt_info_t *opt_info, const TCHAR *opt)
             return DR_FAILURE;
         }
 
-        len = MIN(DR_MAX_OPTIONS_LENGTH - 1, _tcslen(opt));
+        len = strnlen(opt, DR_MAX_OPTIONS_LENGTH);
```

Comment:
opt is type TCHAR which is wchar_t for UNICODE so this is not correct.  Ditto below.

Reply:
I've used the preprocessor to set this to the platform specific "nlen" function.

Done